### PR TITLE
Fix sso podcount alert

### PR DESCRIPTION
**Pull request by matskiv**
## Additional Information
Follow up on #1276 

## Verification Steps
As the verifier of the PR the following process should be done:

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 

Logs: https://gist.githubusercontent.com/matskiv/483eab5330d4a3c99d18fa69dcef017c/raw/d6bb11d7164f37ecc6c6274e912b8a6e697f7ea1/install.log

### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

1. Open Prometheus alerts page
2. Find "SSOPodCount" alert and verify it's not firing

## Checklist

- [ ] Updated [Changelog](https://github.com/integr8ly/installation/blob/master/CHANGELOG.md)
- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

Is an upgrade task required and are there additional steps needed to test this?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: roles/middleware_monitoring_config/tasks/upgrade.yml [hunk 1]
```diff
@@ -1,8 +0,0 @@
----
-# Apply latest CR definitions
-- include_tasks: ./create_alerts.yml
-- include_tasks: ./create_grafanadashboards.yml
-- include_tasks: ./create_blackboxtargets.yml
-- include_tasks: ./prometheus_additional_scrape_config.yml
-- include_tasks: ./patch_blackboxtargets_config.yml
-  when: not eval_self_signed_certs|bool
```
Comment (austincunningham) [inline roles/middleware_monitoring_config/tasks/upgrade.yml]: What was the reason for removing the upgrade task ?

Comment (matskiv) [reply roles/middleware_monitoring_config/tasks/upgrade.yml]: It is not being included in the upgrade playbook and individual monitoring are already included in the upgrade already.

Comment (austincunningham) [reply roles/middleware_monitoring_config/tasks/upgrade.yml]: :+1:

**Comment by austincunningham** [comment]
/lgtm
need to verify , will do so in the morning

**Comment by austincunningham** [comment]
Verified install
https://issues.redhat.com/secure/attachment/12468900/integreatly-install.log
and changes
image
/approve
/assign @jessesarn

**Pull request merged**